### DISCUSSION TITLE
Daten zurücksetzen

### DISCUSSION THREAD

**Discussion by jblond4711** [tbnobody/OpenDTU#632] [Q&A]
Hi,

ich hatte hier erstmal einen Testaufbau. Da in den nächsten Tagen die „richtigen“ Solar Panele installiert werden möchte ich gern die bisherigen Daten zurücksetzen.
Es geht vor allem um den Gesamtertrag. Gibt es die Möglichkeit? Ich würde nur sehr ungern nochmal alles neu flashen und einrichten.

**Comment by tbnobody** [comment]
Hi,

ein zurücksetzen der Daten ist nicht möglich. Der Gesamtertrag wird fest im Inverter gespeichert (vgl. Kilometer im Auto). Du kannst allerdings in den Inverter Einstellungen einen Offset setzen welcher dann auf die Erträge aufsummiert wird. Dadurch ist es z.B. möglich den Ertrag fürs Webinterface oder MQTT anzupassen.

**Reply by jblond4711** [reply]
Danke erstmal.
Halt mich für blond aber wo finde ich die Einstellung für das setzen des Offset Wertes?

**Reply by tbnobody** [reply]
Einstellungen --> Inverter --> Beim entsprechenden Inverter auf den Stift drücken
